apps/web/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,44 @@
+import NextAuth, { AuthOptions, Profile, Session } from 'next-auth'
```

Comment:
filename is strange, is this correct?

Reply:
Yes

Reply:
https://next-auth.js.org/getting-started/example#add-api-route